roadmap.md

```diff
@@ -0,0 +1,52 @@
+# Roadmap
```

Comment:
This roadmap to me is a bit confusing, it more just specifies generic ideas rather than the order of how these things are going to be implemented. I would be more interested to see how each of these things will integrate with Orkestra and in what order.